src/os/bluestore/bluestore_tool.cc

```diff
@@ -582,7 +581,7 @@ int main(int argc, char **argv)
       cerr << "must specify bluestore path" << std::endl;
       exit(EXIT_FAILURE);
     }
-    if (really.empty() || strcmp(really.c_str(), "--yes-i-really-really-mean-it") != 0) {
+    if (!vm.count("yes-i-really-really-mean-it")) {
```

Comment:
Partial matching of long options will result in passing the check, for example, I intentionally entered `--yes`, this  also pass the check.